.github/workflows/codeception.yml

```diff
@@ -16,7 +16,7 @@ env:
     PIMCORE_PROJECT_ROOT: ${{ github.workspace }}
     APP_ENV: test
     PIMCORE_TEST: 1
-#    PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test"
+    PIMCORE_TEST_DB_DSN: "mysql://root@127.0.0.1:33006/pimcore_test"
```

Comment:
@aryaantony92 do we really need to redeclare this env var?

Reply:
or just remove the one with `server_version` since we are not using the matrix value anymore

Reply:
Remove the env with server_version in [c0dd95a](https://github.com/pimcore/customer-data-framework/pull/502/commits/c0dd95a6eaae5c8b112b24196c8505b313c11e12)